examples/plot_quickstart.py

```diff
@@ -92,3 +92,8 @@
     title="Show count metric in pie chart",
 )
 fig[0][0].figure.savefig("filename.png")
+
+# sphinx_gallery_start_ignore
+import os
```

Comment:
Instead of this code in the notebook, perhaps adding an exception for 'filename.png' in `.gitignore` is simpler?

Reply:
Yep, I think that will be better.